packages/@lwc/rollup-plugin/package.json

```diff
@@ -16,7 +16,10 @@
     },
     "dependencies": {
         "@lwc/module-resolver": "1.6.1",
-        "@rollup/pluginutils": "~3.0.9"
+        "@rollup/pluginutils": "~3.0.10"
+    },
+    "peerDependencies": {
+        "rollup": "^1.2.0||^2.0.0"
```

Comment:
Adding missing `rollup` peer dependency. Since the LWC rollup plugin long time stable APIs, we are using the same peer dependency requirements expressed by `@rollup/pluginutils`. This change can go into a patch and don't need the release of a new minor IMO.